gusto/timeloop.py

```diff
@@ -152,6 +161,14 @@ def run(self, t, tmax, pickup=False):
 
 class AdvectionTimestepper(BaseTimestepper):
 
+    def __init__(self, state, advection_dict, diffusion_dict, physics_list=None):
+
+        super(AdvectionTimestepper, self).__init__(state, advection_dict, diffusion_dict)
+        if physics_list is not None:
+            self.physics_list = physics_list
+        else:
+            self.physics_list = []
+
     def run(self, t, tmax, x_end=None):
         state = self.state
 
```

Comment:
I note this isn't your code, but below the `print ...` line should be replaced by proper logging (just to note for later).